ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -278,6 +278,8 @@ Control {
                             spacing: 8
                             Loader {
                                 active: root.messageDetails.messageText !== ""
+                                        // Exclude "placeholder" text cos mobile app send it instead of empty string
+                                        && root.messageDetails.messageText != "<p>placeholder</p>"
```

Comment:
The backend should handle that IMO. We can replace `<p>placeholder</p>` with `""` since it makes more sense

Reply:
Reminder to handle that in the backend. like in the module, you can check the text and if it is `== EMPTY_IMAGE_MESSAGE`, replace with empty string. That way, the frontend doesn't change.